src/hooks/useSubmitEmailForm.ts

```diff
@@ -18,7 +18,12 @@ export const useSubmitEmailForm = (
   const submitForm = React.useCallback(
     (fields: IField[]) => {
       if (!validateEmail(fields[0].value)) {
-        return setResponse({ success: '', error: 'Please enter your work email' });
+        const ind: number = fields[0].value.split('').indexOf('@');
```

Comment:
I don't think you need to split on the empty string. `indexOf` should give you the correct character location in the string:

```suggestion
        const ind: number = fields[0].value.indexOf('@');
```

Or

```suggestion
        const blockedEmail = fields[0].value.split('@')[1];
```